filters/map_composition.py

```diff
@@ -79,82 +86,139 @@ def responseComplete(self):
             else:
                 remove_qml = False
 
-            if exists(project_path):
-                if not overwrite:
-                    msg = 'PROJECT is already existing : %s \n' % project_path
-                    request.appendBody(msg)
-                    return
-                else:
-                    remove(project_path)
+            if exists(project_path) and overwrite:
```

Comment:
The changes now:
1. If overwrite is specified, then delete qgs project file and create new later
2. If project exists and not overwrite, then update layer registry